### PR TITLE
Update foreman_ansible to 1.4.4 [rpm]

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.14
* [x] 1.13
* [ ] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Thanks! Depends on https://github.com/theforeman/foreman-packaging/pull/1522

### CODE REVIEW

**Comment by dLobatog** [comment]
Asset precompilation is now added and assets should show up in the files section now. Thanks @domcleal

File: rubygem-foreman_ansible/rubygem-foreman_ansible.spec [hunk 1]
```diff
@@ -32,7 +32,7 @@ BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.1
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) < 0.9.0
 BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 0.8.14
 BuildRequires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
-BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 1.0.0
+BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 1.1.0
 BuildRequires: %{?scl_prefix}rubygem(foreman_ansible_core) < 2.0.0
 BuildRequires: foreman-plugin >= 1.12.0
```
Comment (domcleal) [inline rubygem-foreman_ansible/rubygem-foreman_ansible.spec]: Missing a BR on foreman-assets at least, please check it builds.

**Comment by dLobatog** [comment]
@domcleal Sorry - just built it with mock, it seems fine (it was just missing the BR for assets)

**Comment by domcleal** [comment]
Merged and building, thanks @dLobatog.

**Pull request merged**